centraldogma/src/test/java/com/linecorp/decaton/centraldogma/CentralDogmaPropertySupplierTest.java

```diff
@@ -133,4 +144,106 @@ public void testGetPropertyAbsentName() {
         PropertyDefinition<Object> missingProperty = PropertyDefinition.define("absent.value", Long.class);
         assertFalse(supplier.getProperty(missingProperty).isPresent());
     }
+
+    @Test
+    public void testRegisterWithDefaultSettings() {
+        when(centralDogma.normalizeRevision(PROJECT_NAME, REPOSITORY_NAME, Revision.HEAD)).thenReturn(
+                CompletableFuture.completedFuture(Revision.HEAD)
+        );
+        when(centralDogma.listFiles(PROJECT_NAME, REPOSITORY_NAME, Revision.HEAD, FILENAME)).thenReturn(
+                CompletableFuture.completedFuture(Collections.emptyMap())
+        );
+        when(centralDogma.push(
+                PROJECT_NAME, REPOSITORY_NAME, Revision.HEAD,
+                String.format("[CentralDogmaPropertySupplier] Property file created: %s",
+                              FILENAME),
+                Change.ofJsonUpsert(FILENAME, defaultProperties()))
+        ).thenReturn(
+                CompletableFuture.completedFuture(
+                        new PushResult(Revision.HEAD, 1)
+                )
+        );
+
+        CentralDogmaPropertySupplier.register(centralDogma, PROJECT_NAME, REPOSITORY_NAME, FILENAME);
+
+        verify(centralDogma, times(1)).normalizeRevision(PROJECT_NAME, REPOSITORY_NAME, Revision.HEAD);
```

Comment:
I think what we wanna test here is, if calling `register()` actually tries to create an expected file with expected content, so the only check against `push()` is meaningful to be testing subject?

Reply:
I deleted the verifying lines: https://github.com/line/decaton/pull/134/commits/61cea291f0dc7c04ef0885a1dea607fb71959819